src/encryption/algorithms.rs

```diff
@@ -1131,9 +1131,11 @@ impl PasswordAlgorithm {
         let mut key = [0u8; 32];
         key.copy_from_slice(file_encryption_key);
 
-        Aes256EbcDec::new(&key.into())
-            .decrypt_block_mut(&mut bytes.into());
-
+        let mut decryptor = Aes256EbcDec::new(&key.into());
+        for block in bytes.chunks_exact_mut(16) {
```

Comment:
I forgot the block size for AES-256 EBC is also 16 bytes and not 32 bytes, so this change looks good.

Reply:
Actually, upon reviewing this again. This doesn't fix anything. The bytes block is only 16 bytes, so we don't necessarily need to iterate over the blocks (other than to make it look consistent).